ui/app/styles/utils/_size_variables.scss

```diff
@@ -23,15 +17,14 @@ $size-9: 0.75rem; // 12px same as $spacing-s
 $size-10: 0.5rem; // 8px same as $spacing-xs
 $size-11: 0.25rem; // 4px same as spacing-xxs
 
-/* Spacing */
+/* Spacing vars in px */
 $spacing-xxs: 4px;
 $spacing-xs: 8px;
 $spacing-s: 12px;
 $spacing-m: 16px;
 $spacing-l: 24px;
 $spacing-xl: 36px;
 $spacing-xxl: 48px;
-$spacing-xxxl: 64px;
```

Comment:
i created this var hoping it would account for the widely used 4rem. It doesn't work that way. 4rem does not === 64 px (16px for 1 rem)